packages/libs/chainweb-node-client/src/send.ts

```diff
@@ -1,7 +1,6 @@
-import type { Base16String, ICommand } from '@kadena/types';
-
 import { parseResponse } from './parseResponse';
 import { stringifyAndMakePOSTRequest } from './stringifyAndMakePOSTRequest';
+import { SendResponse, Base16String, ICommand } from '@kadena/types';
 
```

Comment:
Would it be an idea to have SendResponse typed from chainweb-node-client instead of @kadena/types

Reply:
Done